tests/golden/find-command/find-command.bats

```diff
@@ -18,8 +18,8 @@ load '../helpers'
 /
   secrets/
     google - [2000-01-01 01:01:01]
-      email: johndoe@gmail.com [2000-01-01 01:01:01]
-      pw: [private]            [2000-01-01 01:01:01]
+      email: johndoe@gmail.com$reset [2000-01-01 01:01:01]
+      pw: [private]$reset            [2000-01-01 01:01:01]
```

Comment:
Hmm, maybe appending the reset sequence to _all_ field is a bit too cumbersome, it affects the output of all tests even where there's no need to.

What if we try to make the `Buildable` instance a bit smarter? E.g. append the reset sequence _only when_ the string contains `\x1b[` (the sequence for [Control Sequence Introducers](https://en.wikipedia.org/wiki/ANSI_escape_code#CSIsection))

Reply:
Done